spring-batch-docs/modules/ROOT/pages/readers-and-writers/item-reader-writer-implementations.adoc

```diff
@@ -7,19 +7,17 @@ discussed in the previous sections.
 [[decorators]]
 == Decorators
 
-In some cases, a user needs specialized behavior to be appended to a pre-existing
-`ItemReader`.   Spring Batch offers some out of the box decorators that can add
-additional behavior to to your `ItemReader` and `ItemWriter` implementations.
-
-Spring Batch includes the following decorators:
+In some cases, you might need specialized behavior to be appended to a pre-existing
+`ItemReader` or `ItemWriter` implementation.
+For this purpose, Spring Batch offers the following out-of-the-box decorators:
```

Comment:
I made the intro more concise, but I can revert it if you prefer the previous version.